field/result_field_test.go

```diff
@@ -144,12 +144,12 @@ func (*testResultFieldSuite) TestMain(c *C) {
 
 	// For GetFieldIndex
 	f1 := &field.Field{
-		Expr: &expression.Ident{CIStr: model.NewCIStr("c1")},
-		Name: "a",
+		Expr:   &expression.Ident{CIStr: model.NewCIStr("c1")},
```

Comment:
Value is kind of weird, Maybe we should change its function to use pointer.  
Expr:   expression.Value{Val: "c1+1"},